### PR TITLE
Qprof: Display plans in multiple sections

### PR DESCRIPTION
**Pull request by oualib**
- closes JIRA: https://jira.verticacorp.com/jira/browse/VER-93016

### CODE REVIEW

File: verticapy/performance/vertica/qprof.py [hunk 1]
```diff
@@ -2139,6 +2138,71 @@ def get_qsteps(
...
+            AND step_label ILIKE '%TempRelWrite%' 
+            ORDER BY step_key ASC;"""
+        query = self._replace_schema_in_query(query)
+        res = _executeSQL(
+            query,
+            title="Getting the corresponding query",
+            method="fetchall",
+        )
+        return list(dict.fromkeys([q[0] for q in res]))
+
```
Comment (naksman) [inline verticapy/performance/vertica/qprof.py:2205]: What will happen with more complicated TempRel queries, will their plan trees be displayed horizontally all the way to the right? Is there a limit on how wide the horizontal display is?

**Review by mail4umar — APPROVED** [review]

File: verticapy/performance/vertica/qprof.py (hunk 1 above)
Comment (oualib) [reply verticapy/performance/vertica/qprof.py:2205]: yes, it is displayed horizontally. There is no limit.

**Comment by naksman** [comment]
Is everything all set? I don't see Approve button for myself, I don't need to approve as it's already approved, right?

**Comment by oualib** [comment]
> Is everything all set? I don't see Approve button for myself, I don't need to approve as it's already approved, right?

you click on "file changed", you look at each file and give comment if needed. Then you can check the small checkbox and approve.

**Review by naksman — APPROVED** [review]

**Pull request merged**